app/window.go

```diff
@@ -192,6 +192,10 @@ func (w *Window) validateAndProcess(frameStart time.Time, size image.Point, sync
 					w.ctx, err = d.NewContext()
 				})
 				if err != nil {
+					if errors.Is(err, gpu.ErrDeviceLost) {
```

Comment:
This change shouldn't be necessary if NewContext never returns ErrDeviceLost, as commented above.